### DISCUSSION TITLE
git bash

### DISCUSSION THREAD

**Discussion by nandini965** [learndevopsonline/learndevopsonline#655] [Q&A] [answered]
iam using aws cloud shell what should i install  in git bash
![Screenshot (21)](https://user-images.githubusercontent.com/129321520/233698953-cdd5115b-1382-4f09-814a-1a9dbade1430.png)

**Comment by noolivara** [comment, marked as answer]
@nandini965 ,AWS Cloud Shell comes with Git pre-installed. You can check if Git is installed by opening the terminal and running the following command.

**git --version**

**Comment by nandini965** [comment]
Okay thanq
